Assets/Scripts/Combat/Hitbox.cs

```diff
@@ -10,14 +10,17 @@ public class Hitbox : DisableSpriteRender
     public bool isEnemyHitbox = false;
     public bool hitOnce = false;
 
+    [SerializeField] PossessionManager possessionManager;
```

Comment:
Nit (minor): this variable is named the same as the [out variable on line 27](https://github.com/MitchellCoon/MetroidvaniaGameJamFeb23/pull/45/files#diff-e46262b23de53ed8582a70b8be4c5a060b9595f97b168768c795e3386ecb9b22L24) and introduces variable shadowing - could we name this something different? Even `_posessionManager` with an underscore would be enough.

Reply:
I think it would make sense to have one as the sourcePossessionManager and the other as the targetPossessionManager